measures/measures-schema.yaml

```diff
@@ -197,3 +205,55 @@ definitions:
       - thoracicSurgery
       - urology
       - vascularSurgery
+
+  eligibilityOption:
+    type: object
+    properties:
+      submissionMethods: { $ref: #/definitions/qualityCodesSubmissionMethods }
+      sexCode:
+        enum: [M, F]
+      minAge:
+        type: integer
+      maxAge:
+        type: integer
+      diagnosisCodes: { $ref: #/definitions/arrayOfStringIdentifiers }
+      additionalDiagnosisCodes: { $ref: #/definitions/arrayOfStringIdentifiers }
+      procedureCodes:
+        type: array
+        items: { $ref: #/definitions/codeObject }
+    required: [submissionMethods]
```

Comment:
They are required in any `any of` type of relationship, e.g. procedureCodes or diagnosisCodes may present a valid eligibility option, updated the implementation here: https://github.com/CMSgov/qpp-measures-data/pull/28/commits/c26c9011f176c82cc4be7cdc8aa22b3c17d8a87a